### ISSUE TITLE
Cannot create macOS 15.2 VMs 

### ISSUE THREAD

**Issue by natewalck** [cirruslabs/tart#929]
When trying to make a VM using 15.2b1 (`UniversalMac_15.2_24C5057p_Restore.ipsw`), I get the following error:

```
tart create 15.2.0-test01 --from-ipsw ./UniversalMac_15.2_24C5057p_Restore.ipsw --disk-size 128
Installing OS...
0%
Error: An error occurred during installation. Installation failed.
```

i'm on 15.2 (`24C5057p`) and I installed the Xcode tools for 16.1 to see if that would help, but alas it did not. I also rebooted.

I seem to remember this happening in the past and newer xcode tools generally resolved it, but that wasn't the case this time.

**Comment by natewalck** [comment]
In the release notes for this: 

```
Known Issues in macOS 15.2 Beta
• Installation of macOS Sequoia 15.2 Beta in VMs is not available.
```

I guess we wait until beta 2.

**Comment by 1stqualifier** [comment]
Try running it without the --disk-size parameter and see if it works. I tried installing Sonoma from ipsw and get the same error when specifying --disk-size. Not sure if it worked before, i.e. if you can change the disk size from the default 50 at the install stage. Increasing it afterwards is a royal pain in the backside.

**Comment by natewalck** [comment]
> Try running it without the --disk-size parameter and see if it works. I tried installing Sonoma from ipsw and get the same error when specifying --disk-size. Not sure if it worked before, i.e. if you can change the disk size from the default 50 at the install stage. Increasing it afterwards is a royal pain in the backside.

Hot damn, that worked. Thanks for mentioning this!

**Comment by 1stqualifier** [comment]
> > Try running it without the --disk-size parameter and see if it works. I tried installing Sonoma from ipsw and get the same error when specifying --disk-size. Not sure if it worked before, i.e. if you can change the disk size from the default 50 at the install stage. Increasing it afterwards is a royal pain in the backside.
> 
> Hot damn, that worked. Thanks for mentioning this!

No worries, I raised this here https://github.com/cirruslabs/tart/issues/934 since I came across this issue myself as 50 doesn't cut it anymore, let's see if there's some official guidance on how to use it correctly or if this functionality is broken.

**Comment by edigaryev** [comment]
Seems to be fixed in the latest [macOS Sequoia 15.2 Beta 4](https://developer.apple.com/documentation/macos-release-notes/macos-15_2-release-notes#Virtualization):

>Fixed: Installation of macOS Sequoia 15.2 Beta in VMs is not available. (137528160)
